src/Persistent/Pure.hs

```diff
@@ -542,7 +556,7 @@ moveIdeaToTopic ideaId mTopicId =
 setTopicPhase :: AUID Topic -> Phase -> AUpdate ()
 setTopicPhase tid phase = withTopic tid . topicPhase .= phase
 
-addTopic :: Timestamp -> AddDb Topic
```

Comment:
Also, this should keep the same type, and we should add a function `addTopicYieldLocs`.  That will make lots of the follow-up changes unnecessary.

Thanks, @andorp for the idea!